### DISCUSSION TITLE
Mpdf.php needs minor change to avoid 'deprecated' warning in PHP 8.1

### DISCUSSION THREAD

**Discussion by wetken** [mpdf/mpdf#1815] [General]
~/vendor/mpdf/mpdf/src/Mpdf.php line 6560 needs to read

if ($blockdir == 'ltr' && isset($lastfontreqstyle) && strpos($lastfontreqstyle, "I") !== false && strpos($lastfontstyle, "I") === false) { // Artificial italic

to avoid strpos deprecated message in PHP 8.1

**Comment by finwe** [comment]
This is not the content of line 6560 in the last commit of the development version.

If you are getting deprecated errors in the last version, create an Issue with reproducible code example.
